src/ReportPortal.Shared/Extensibility/Embedded/Analytics/AnalyticsReportEventsObserver.cs

```diff
@@ -24,18 +25,10 @@ public class AnalyticsReportEventsObserver : IReportEventsObserver, IDisposable
 
         private readonly string _platformVersion;
 
-        private readonly HttpClient _httpClient;
+        private static HttpClient _httpClient;
```

Comment:
Do not make it static for now.

Reply:
It was static, so that we could have the one instance per application scope.
But, given how this observer is loaded, we effectively have that anyway. 👍